rocketry/_base.py

```diff
@@ -1,8 +1,12 @@
-from typing import TYPE_CHECKING
+from typing import TYPE_CHECKING, Any, ClassVar
+from pydantic.dataclasses import dataclass, Field
+from pydantic import BaseModel
 
 if TYPE_CHECKING:
     from rocketry import Session
 
-class RedBase:
+class RedBase():
```

Comment:
I think the brackets are not needed here.

Reply:
Agreed, left in on accident and will be removed on next commit.